CommonTools/Utils/interface/ExpressionEvaluatorTemplates.h

```diff
@@ -23,8 +24,9 @@ namespace reco {
   template<typename Object>
   struct ValueOnObject {
     virtual double eval(Object const&) const =0;
+    virtual ~ValueOnObject(){};
```

Comment:
in this file I also see CutOnObject and SelectIndecesInCollection with virtual methods and no virtual destructor defined.
Why?
Are these not used anywhere yet to trigger the warning?

It looks like these other types should have a destructor defined as well.